### ISSUE TITLE
Move a11y components to a new @Yoast/a11y package

### ISSUE THREAD

**Issue by bintzandt** [Yoast/javascript#34]
## Summary
We want to create a new `@Yoast` NPM package with all our a11y components in it. These components should be put in `@Yoast/a11y/components`. In the future our a11y helpers and other functions can also be moved to this package so that we offer one package that covers all (or most of) our a11y needs.

## Explanation
_Separate all the things!_ Offering a separate a11y components package ensure that it will be easier to reuse these components in our existing projects.
